### ISSUE TITLE
How to get backend running?

### ISSUE THREAD

**Issue by Thesdam** [silveredgold/beta-censoring#30]
Hi, newbie here!

I just have absolutely no idea how to get the backend running. Nothing within the beta-censoring-main folder appears 'runnable' to me and I feel like I've tried everything.

I've followed the user guides extensively and get nowhere. Can I please get this explained in layman's terms? 

Thank you so much!

**Comment by silveredgold** [comment]
If you've downloaded `beta-censoring-main`, you didn't follow the guides because you downloaded the wrong thing
